code/__defines/damage_organs.dm

```diff
@@ -35,6 +35,7 @@
 #define O_KIDNEYS  "kidneys"
 #define O_APPENDIX "appendix"
 #define O_CELL     "cell"
+#define O_POSIBRAIN	"posibrain"
```

Comment:
That define use only with species what not used here.
Is that really related to Preference?

Reply:
it used in preference and since we plan to add species why not add it now?

Reply:
> since we plan to add species

But we're not planning that